### PR TITLE
fixes KA shakes

### PR DESCRIPTION
**Pull request by Time-Green**
:cl:
fix: Fixes the kinetic accelerator shaking with a wideguard
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/energy/kinetic.dm [hunk 1]
```diff
@@ -270,11 +270,11 @@
 	rank = "special"
 
 /obj/item/kinetic_part/grip/wide/attach(obj/item/weapon/gun/energy/kinetic_accelerator/KA,mob/user)
-	KA.weapon_weight = WEAPON_MEDIUM
+	KA.w_class = 4
```
Comment (Kmc2000) [inline code/modules/projectiles/guns/energy/kinetic.dm:273]: Why did you change it to 4 and not another define? looks weird

File: code/modules/projectiles/guns/energy/kinetic.dm [hunk 2]
```diff
@@ -270,11 +270,11 @@
...
 	KA.trigger_guard = TRIGGER_GUARD_ALLOW_ALL
 
 /obj/item/kinetic_part/grip/wide/detach(obj/item/weapon/gun/energy/kinetic_accelerator/KA,mob/user)
-	KA.weapon_weight = WEAPON_LIGHT
+	KA.w_class = 3
```
Comment (Kmc2000) [inline code/modules/projectiles/guns/energy/kinetic.dm:277]: Again here...

File: code/modules/projectiles/guns/energy/kinetic.dm [hunk 3]
```diff
@@ -270,11 +270,11 @@
 	rank = "special"
 
 /obj/item/kinetic_part/grip/wide/attach(obj/item/weapon/gun/energy/kinetic_accelerator/KA,mob/user)
```
Comment (Kmc2000) [inline code/modules/projectiles/guns/energy/kinetic.dm:272]: This doesn't return ..() , does this code work? I.E was it tested?. Looks like the grip just applies magic to it and is never deleted as an item.

File: code/modules/projectiles/guns/energy/kinetic.dm (hunk 2 above)
Comment (Time-Green) [reply code/modules/projectiles/guns/energy/kinetic.dm:277]: Couldn't find any defines. They were all integers

File: code/modules/projectiles/guns/energy/kinetic.dm (hunk 3 above)
Comment (Kmc2000) [reply code/modules/projectiles/guns/energy/kinetic.dm:272]: Feel free to correct me on that front

Comment (Time-Green) [reply code/modules/projectiles/guns/energy/kinetic.dm:272]: switchPart() does the actual attaching and de-attaching. Line 56
I prefer to split my functions, that way I don't need to constantly use ..() and it makes the code easier to work with

Comment (Kmc2000) [reply code/modules/projectiles/guns/energy/kinetic.dm:272]: Ahh alright

File: code/modules/projectiles/guns/energy/kinetic.dm (hunk 2 above)
Comment (ShadowDeath6) [reply code/modules/projectiles/guns/energy/kinetic.dm:277]: look at backpacks or something, like w_class = w_class_normal

Comment (ShadowDeath6) [reply code/modules/projectiles/guns/energy/kinetic.dm:277]: I was wrong actually, weird

**Pull request closed without merging**